test/private-api/alias.e2e-spec.ts

```diff
@@ -156,7 +156,7 @@ describe('Alias', () => {
           .put(`/api/private/alias/${forbiddenNoteId}`)
           .set('Content-Type', 'application/json')
           .send(changeAliasDto)
-          .expect(400)
+          .expect(404)
```

Comment:
404 when requesting a forbidden ID is also weird. IMO it should return `Forbidden`.

Reply:
In this case 403 Forbidden seems right.

But speaking in strict terms it's "updating" and not "requesting" a forbidden alias.
For *requesting* a forbidden alias, 404 Not Found seems right.